pkg/route/api/validation/validation.go

```diff
@@ -37,11 +37,16 @@ func ValidateRoute(route *routeapi.Route) field.ErrorList {
 		result = append(result, field.Invalid(specPath.Child("path"), route.Spec.Path, "passthrough termination does not support paths"))
 	}
 
-	if len(route.Spec.To.Name) == 0 {
-		result = append(result, field.Required(specPath.Child("to", "name"), ""))
+	if len(route.Spec.To) == 0 {
```

Comment:
Limit the number of alternates to something reasonable, like 3.  I don't want people abusing this feature until we understand it.